projects/netgrif-components-core/src/lib/resources/interface/transition-import.ts

```diff
@@ -0,0 +1,8 @@
+import {Position} from './position';
+
+export interface TransitionImport {
+    importId: string;
+    position: Position;
+    stringId: string;
+    title: object;
```

Comment:
title: object ?

Reply:
its I18NString, but i think we dont have an interface for it, and sometimes its only string and not i18nstring